py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -19,7 +20,8 @@ class BatchArgs(NamedTuple):
     s3_prefix: str
     s3_bucket: str
     filesize_threshold: int
-    pretty: bool
+    print_events: bool = False
+    dry_run: bool = False
```

Comment:
i decided we should always log out the stuff that was behind pretty. the print events is used for local testing and the dry run stops us from triggering the lambda (which will raise an exception if the lambda arn isn't defined)